Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -46,7 +50,7 @@ CheckSource()
 	else
 		__file_content=$(cat $current_clocksource)
 		LogMsg "Found currect_clocksource $__file_content in $current_clocksource"
-		if [[ $__file_content == *"$clocksource"* ]]; then
+		if [[ $__file_content == "$clocksource" ]]; then
```

Comment:
This will add bring back the old test results.  I am not sure this is correct.

Reply:
Earlier we used to have short name like (hv_*). Now full name is used to avoid pattern matching.